### DISCUSSION TITLE
Board/Rev0.3/pico-ice.kicad_sch - 2022/06/24

### DISCUSSION THREAD

**Discussion by josuah** [tinyvision-ai-inc/pico-ice#7] [Design Review]
https://github.com/tinyvision-ai-inc/pico-ice/blob/main/Board/Rev0/For%20Review/pico-ice_r0.3.pdf

**Comment by josuah** [comment]
**TODO: Add a diode to prevent backpowering the USB?**

**Reply by josuah** [reply]
How would this occur? In case there is another power source through the board?

In which case, would this be for when the host would provide fewer power than the external power source?

**Reply by vr2045** [reply]
This could happen if the user supplied 5V from the J2(SPARE) header and had a USB cable plugged in.
